### PR TITLE
fix typos in c documentation

### PR DESCRIPTION
**Pull request by garanews**
fix typos in c documentation

### CODE REVIEW

File: content/c/concepts/variables/variables.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Variables'
-Description: 'A variable is a location in computer memory used to store data, usually for use in a program. To declare a variable in C, the type of variable is first included, followed by the name, and the value. pseudo type name = value;  Though if a variable does not yet need a value assigned to it, this can be ommitted and assigned later. c'
+Description: 'A variable is a location in computer memory used to store data, usually for use in a program. To declare a variable in C, the type of variable is first included, followed by the name, and the value. pseudo type name = value;  Though if a variable does not yet need a value assigned to it, this can be omitted and assigned later. c'
```
Comment (SSwiniarski) [inline content/c/concepts/variables/variables.md:3]:
```suggestion
Description: 'A variable is a location in computer memory used to store data.'
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@garanews, I did a review and left one comment.

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@garanews Let's **bold** the concept "variable" on line # 15 in `variables.md`. That's all from me. 😄

cc: @SSwiniarski anything I might have missed?

**Review by Dusch4593 — APPROVED** [review]
@garanews LGTM! 👍🏻

I made the following edits in 1e22e811dbafb8555ee7f88195ba77989fc1cdf8:

- **Bolded** the first mention of the concept "variables" in `variables.md`.
- Light rephrasing/rearranging in the Syntax section including a new \```pseudo code block.

**Review by SSwiniarski — APPROVED** [review]
Looks good to merge

**Pull request merged**